grin/src/server.rs

```diff
@@ -124,6 +124,7 @@ impl Server {
 					LOGGER,
 					"No seed configured, will stay solo until connected to"
 				);
+				seed.monitor_only(evt_handle.clone());
```

Comment:
We want the server to monitor peers even if we start without any seeds (somebody else may communicate with us).